test/e2e/cypress/integration/clusters_overview.js

```diff
@@ -44,5 +44,226 @@ context('Clusters Overview', () => {
         cy.get(`a:contains(${clusterID})`).should('be.visible');
       });
     });
+
+    describe('Health status for each cluster is correct', () => {
+      const healthyClusterScenario = {
```

Comment:
This data looks something that should go in the fixtures, in `clusters-overview` folder I guess